components/atom/tooltip/src/index.scss

```diff
@@ -0,0 +1,149 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$fz-atom-tooltip: $fz-m !default;
+$lh-atom-tooltip: $lh-m !default;
+$wlc-atom-tooltip: 4 !default; // -webkit-line-clamp → lines to show
+
+$maw-atom-tooltip: 250px !default;
+$mah-atom-tooltip: 100px !default;
+$c-atom-tooltip: $c-white !default;
+$bgc-atom-tooltip: $c-black !default;
+$bdrs-atom-tooltip: 4px !default;
+$op-atom-tooltip: 1 !default;
+$pt-atom-tooltip: $p-m !default;
+$pb-atom-tooltip: $p-m !default;
+$pl-atom-tooltip: $p-m !default;
+$pr-atom-tooltip: $p-m !default;
+$m-atom-tooltip: 0 !default;
+$z-atom-tooltip: 1070 !default;
+
+$w-atom-tooltip-arrow: .8rem !default;
+$h-atom-tooltip-arrow: .4rem !default;
+$c-atom-tooltip-arrow: $bgc-atom-tooltip !default;
+
+// Base class
+.sui-AtomTooltip {
+  display: block;
+  font-size: $fz-atom-tooltip;
+  margin: $m-atom-tooltip;
+  opacity: 0;
```

Comment:
Are we rendering all instances of tooltip in the DOM even when not displayed ?

Reply:
The tooltip markup is added and removed from the DOM every time the `isOpen` state change, so yes, the `opacity` values can be removed